tests/parsers/agilent_gen5/to_allotrope_test.py

```diff
@@ -48,7 +50,8 @@ def test_to_allotrope_absorbance(filename: str) -> None:
     expected_filepath = (
         f"tests/parsers/agilent_gen5/testdata/absorbance/{filename}.json"
     )
-    allotrope_dict = from_file(test_filepath, VENDOR_TYPE)
+    encoding = "UTF-16" if filename == UTF_16_FILENAME else None
```

Comment:
It seems from this like this change is somehow enforcing encoding somewhere where it was previously being auto-detected? 

That makes me a bit worried about pushing this through. This seems like the kind of change that should have no test changes.

Can you tell why this is needing the encoding passed now?

Reply:
it doesn't -- this was me wanting to explicitly test passing an encoding in. This test passes if I revert all the changes here, and I agree that it's more reassuring to make these changes with no changes here. So I'll revert shortly.